daemon/status.go

```diff
@@ -162,10 +162,6 @@ func (d *Daemon) getStatus() models.StatusResponse {
 
 	sr.Cluster = d.getNodeStatus()
 
-	if d.ciliumHealth != nil {
-		sr.Cluster.CiliumHealth = d.ciliumHealth.GetStatus()
```

Comment:
Seems reasonable. I'm not sure whether we want `cilium status` to fail if this fails, though (doesn't really make sense in k8s environments since `cilium-health` would have its own health checks, then in non-k8s environments someone may choose just not to run `cilium-health`)

Reply:
True... I guess this also means we might need to change docs that talk about cilium-health.

Reply:
So let me understand your suggestion. You want to:

- keep the information about `cilium-health` in `cilium status` (just to make it easier for people using Cilium with systemd, without Kubernetes)
- but return status 0 even if `cilium-health status` fails

Is that correct?

Reply:
> return status 0 even if `cilium-health status` fails

Return status 0 from `cilium status` if its call to `cilium-health status` fails. The output should still print that it was unable to connect to `cilium-health` with a user-friendly string describing that this may or may not represent a real issue.